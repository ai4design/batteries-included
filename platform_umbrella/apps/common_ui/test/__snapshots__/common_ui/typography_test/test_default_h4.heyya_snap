<h4 class="text-lg sm:text-xl uppercase text-semibold">
  Test H4 Accent Color Header
</h4>