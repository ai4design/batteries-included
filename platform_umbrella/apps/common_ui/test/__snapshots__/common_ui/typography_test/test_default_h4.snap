"<h4 class=\"text-lg font-bold leading-loose text-blizzard-blue-800\">\n  Test H4 Accent Color Header\n</h4>"